.github/workflows/build.yaml

```
@@ -11,17 +11,18 @@ jobs:
       - name: Checkout Packer project
         uses: actions/checkout@v2
 
-      - name: Packer Init
+      - name: Export Application Credentials
         run: |
-          curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
-          sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
-          sudo apt-get update && sudo apt-get install packer
-          packer init .
-
-      - name: Build Packer Artifact
+          echo ${PACKER_SA_KEY} > $GOOGLE_APPLICATION_CREDENTIALS
```

@obasekiosa you can make this a oneliner step.
```suggestion
         - name: Export Application Credentials
           run: echo ${{ secrets.PACKER_SA_KEY }} > $GOOGLE_APPLICATION_CREDENTIALS
```
and then remove the `PACKER_SA_KEY` env variable from below. You might be able to replace `GOOGLE_APPLICATION_CREDENTIALS` in the script with the proper value `/home/runner/packer_sa_key.json`, and the remove the var entirely, but let see if this var is needed for the build job.

---

`GOOGLE_APPLICATION_CREDENTIALS` env variable is needed if running the googlecompute plugin (see [here](https://github.com/hashicorp/packer-plugin-googlecompute/blob/main/docs/builders/googlecompute.mdx#running-outside-of-google-cloud))

But I'll change `line 16` to be: 
```yaml
echo ${{ secrets.PACKER_SA_KEY }} > $GOOGLE_APPLICATION_CREDENTIALS
```

Then, I'll delete the `PACKER_SA_KEY` var